vignettes/VCF.Rmd

```diff
@@ -15,7 +15,11 @@ knitr::opts_chunk$set(
 )
 ```
 
-## Call required libraries. Initialize gosling with `use_gosling()`
+This tutorial walks through the process of creating a Circos plot using VCF (Variant Call Format) data, with a focus on utilizing the shiny.gosling package to achieve interactive visualizations within a Shiny app.
+
+## Libraries and Initialization
+
+First, we load the necessary libraries for our analysis, including StructuralVariantAnnotation, ggbio, shiny, and shiny.gosling. We also initialize the shiny.gosling package using the use_gosling() function.
 
 ```{r setup, eval=FALSE, message=FALSE}
 require(StructuralVariantAnnotation)
```

Comment:
As mentioned below, why require instead of library=

Reply:
Updated.